ui/src/containers/SampleQueueContainer.js

```diff
@@ -128,7 +128,6 @@ class SampleQueueContainer extends React.Component {
     return (
       <div style={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
         <QueueControl
-          ref="queueContainer"
```

Comment:
[no-string-refs](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md) — this ref is unused; there are other, more complicated cases that will need to be refactored.